arch/constants.go

```diff
@@ -0,0 +1,9 @@
+// +build !arm
```

Comment:
as discussed, maybe we can put these constants in `/util/architecture/constants.go`